vms/platformvm/txs/convert_subnet_tx.go

```diff
@@ -62,3 +81,43 @@ func (tx *ConvertSubnetTx) SyntacticVerify(ctx *snow.Context) error {
 func (tx *ConvertSubnetTx) Visit(visitor Visitor) error {
 	return visitor.ConvertSubnetTx(tx)
 }
+
+type ConvertSubnetValidator struct {
+	// TODO: Should be a variable length nodeID
+	NodeID ids.NodeID `serialize:"true" json:"nodeID"`
+	// Weight of this validator used when sampling
+	Weight uint64 `serialize:"true" json:"weight"`
+	// Initial balance for this validator
+	Balance uint64 `serialize:"true" json:"balance"`
+	// [Signer] is the BLS key for this validator.
+	// Note: We do not enforce that the BLS key is unique across all validators.
+	//       This means that validators can share a key if they so choose.
+	//       However, a NodeID + Subnet does uniquely map to a BLS key
+	Signer signer.ProofOfPossession `serialize:"true" json:"signer"`
+	// Leftover $AVAX from the [Balance] will be issued to this owner once it is
+	// removed from the validator set.
+	RemainingBalanceOwner message.PChainOwner `serialize:"true" json:"remainingBalanceOwner"`
+	// This owner has the authority to manually deactivate this validator.
+	DeactivationOwner message.PChainOwner `serialize:"true" json:"deactivationOwner"`
```

Comment:
To align with the spec:
```suggestion
	DisableOwner message.PChainOwner `serialize:"true" json:"disableOwner"`
```

Reply:
I resolved this because it isn't really a spec thing (as the serialization isn't impacted by the name). The ACP does call this the `DisableOwner`... But the state package calls it `DeactivationOwner`. I figured if we wanted to do a naming pass we could do that separately.